Probesys/bileto
# imp: Retrieve email requester from Reply-To header

## Related issue(s)



https://github.com/Probesys/bileto/issues/43

## Changes



- improve the MailboxEmailFactory to accept a `replyTo` attribute
- allow to get the `Reply-To` header from MailboxEmail
- consider the requester to be the `Reply-To` address if set

## How to test manually



- send an email to `support@example.com` with a `Reply-To` different from the sender
- check a ticket is created with the address from the `Reply-To`

## Checklist
